dashboards/create-dashboard.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
+
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import os
+import subprocess
+import sys
+
+
+def build_template(dashboard_name, cluster_name, space):
+    # Find the template directory. We assume it is in the same location as the
+    # script. We assume it is in the same location as the script. We assume it
+    # is in the same location as the script. We assume it is in the same
+    # location as the script.
+    dashboard_path = os.path.join(os.path.dirname(os.path.abspath(__file__)), "dashboard-template.json")
+
+    # Replace the sentinel values.
+    with open(dashboard_path) as f:
+        return f \
```

Comment:
maybe leave a note here that this is safe because the values are all guaranteed to be valid DNS labels and thus can't contain a backslash or quote which would escape JSON

Reply:
Done